test/ETO/ETOTerms.js

```diff
@@ -278,7 +308,7 @@ contract("ETOTerms", ([, admin, investorDiscount, investorNoDiscount, ...investo
       await expect(
         redeployTerms({
           MIN_TICKET_EUR_ULPS: modMinTicket,
-          PUBLIC_DISCOUNT_FRAC: Q18.sub(pubPriceFraction).add(1),
+          PUBLIC_DISCOUNT_FRAC: Q18.sub(pubPriceFraction.sub(10)),
```

Comment:
why is it that before we increased the discount to make it fail, and now we decrease it to make it fail?